_includes/contact_form.html

```diff
@@ -352,7 +352,7 @@
   onSubmit="return remapvalues(this); return true;"
   class="usa-form"
 >
-  <input type="hidden" name="retURL" value="http://login.gov" />
+  <input type="hidden" name="retURL" value="https://login.gov{{ '/' | locale_url }}" />
```

Comment:
do we know if the sandbox env is going to look for the federalist URL or some sort of origin header?

Reply:
@zachmargolis I am not sure

Reply:
If we're not able to test this in preview, I'm sure that we could just try deploying it, especially since the repo deploys quickly we can easily undo